CONTRIBUTING.md

```diff
@@ -58,3 +58,25 @@ import Foo as F
 import Qux (Q(..), q)
 import qualified A
 ```
+
+## Development Workflow
+
+Here we outline a general workflow new contributors could adopt for a nice incremental development experience.
+
+The root folder consists of two stack-*.yaml config files.
+
+1. stack-night.yaml - Config file defines all the settings to build and test any changes.
+2. stack-bench.yaml - Used to run benchmark tests.
+
+Therefore, a ghci development experience would be:
+
+1. `stack ghci --stack-yaml stack-nightly.yaml --test`
+2. :r to reload
```

Comment:
```suggestion
2. :r to recompile
```

(the command is called reload but I think this explains what we are after better)